wpstarter/templates/.env.example

```diff
@@ -227,12 +227,10 @@ WP_HOME=
 #WP_PROXY_USERNAME
 #WP_PROXY_PASSWORD
 #WP_PROXY_BYPASS_HOSTS
-#WP_ACCESSIBLE_HOSTS
-#WP_HTTP_BLOCK_EXTERNAL
 
 
 #--------------------------------------------------------------------------------------------------#
 #                                          MISCELLANEOUS                                           #
 #--------------------------------------------------------------------------------------------------#
 #WP_MAIL_INTERVAL=300
-#WP_DEFAULT_THEME=twentyfifteen
\ No newline at end of file
+#WP_DEFAULT_THEME=twentyfifteen
```

Comment:
@GaryJones the minimum required WP version for WP starter is 4.2, where twentysixteen is  not available yet. So this needs to stay twentyfifteen untill the min version is bumped up.

I am ready to merge this, if @franz-josef-kaiser is agree (I saw he self-assigned the issue) we can merge this like now.